spring-boot-app-sso-auth-code-flow/accelerator.yaml

```diff
@@ -0,0 +1,67 @@
+accelerator:
+  options:
+    - name: packageName
+      inputType: text
+      defaultValue: "com.example.customerprofile"
+      label: Application root package
+      required: true
+
+engine:
+  merge:
+    - type: Include
+      patterns: [ "**/profile.html", "**/application-local.yml" ]
+
+    - include: [ "**/OidcUserController.java", "**/WebSecurityConfiguration.java" ]
+      chain:
+        - type: OpenRewriteRecipe
+          recipe: org.openrewrite.java.ChangePackage
+          options:
+            oldPackageName: "'com.vmware.tap.fragments.appssoauthcodeflow'"
+            newPackageName: "#packageName"
+
+    - include: [ "pom.xml" ]
+      chain:
+        - type: ReplaceText
+          substitutions:
+            - text: "<dependencies>"
+              with: |
+                '<dependencies>
```

Comment:
Why not add the 3 dependencies in one go?

Reply:
Done